StortSpelProjekt/Engine/src/AudioEngine/AudioVoice.cpp

```diff
@@ -79,3 +82,37 @@ void AudioVoice::Stop()
         m_pSourceVoice->SubmitSourceBuffer(m_pBuffer);
     }
 }
+
+void AudioVoice::Resume()
+{
+    if (m_Initialized)
+    {
+        // reset the buffer if sound has ended
+        XAUDIO2_VOICE_STATE test;
+        m_pSourceVoice->GetState(&test);
```

Comment:
Please change the variable name from the temporary "test" to something more befitting. voiceState perhaps?